src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -527,6 +527,10 @@ private int getLimit(boolean includeItem) {
       case 404:
         result = Future.succeededFuture(PostPatronAccountItemHoldByIdAndItemIdResponse.respond404WithTextPlain(message));
         break;
+        case 422:
+          final Errors errors = Json.decodeValue(message, Errors.class);
+          result = Future.succeededFuture(PostPatronAccountItemHoldByIdAndItemIdResponse.respond422WithApplicationJson(errors));
+          break;
```

Comment:
Please change the indentation for this case statement to be consistent with the others

Reply:
Done